<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogTabUI</class>
 <widget class="QWidget" name="LogTabUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogTab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="sourceStatusGroupBox">
     <property name="title">
      <string>Source Status</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelSourceType">
          <property name="text">
           <string>Source Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSourceStatus">
          <property name="text">
           <string>Source Status:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelSourceTypeText">
          <property name="text">
           <string>File / RPC Connection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSourceStatusText">
          <property name="text">
           <string>loading/loaded, connected/disconnected</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fileOperationGroupBox">
     <property name="title">
      <string>File Operations</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="saveAllEntriesToFile">
          <property name="text">
           <string>Save Log to File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveVisibleEntriesToFile">
          <property name="text">
           <string>Save visible Log to File</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filterGroupBox">
     <property name="title">
      <string>Filter</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="logLevelComboBox">
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="moduleNameComboBox">
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="threadIDComboBox">
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearFilterPushButton">
        <property name="text">
         <string>Clear Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadFilterButton">
        <property name="text">
         <string>Load Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveFilterButton">
        <property name="text">
         <string>Save Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>411</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="entriesTableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
